src/dashboard/components/Header/index.ts

```diff
@@ -5,7 +5,7 @@ const Header = styled.h1`
     font-weight: 300;
     font-size: 30px;
     line-height: 38px;
-    color: ${({ theme }) => theme.colors.primary};
+    color: ${(props: any) => props.theme.colors.primary};
```

Comment:
```suggestion
    color: ${({ theme }: { theme: typeof defaultTheme }) => theme.colors.primary};
```
might work, assuming that you import `defaultTheme.js` (might need to be converted to TS). https://emotion.sh/docs/typescript#define-a-theme might work to do it more globally.